### ISSUE TITLE
What GPU did this project used in the paper?

### ISSUE THREAD

**Issue by jd-yang-b** [ubisoft/ubisoft-laforge-daft-exprt#8]
Hi, what gpu was used for the experiment in the paper? Like how much of GPU memory is needed?

**Comment by julianzaidi** [comment]
Hi, we used a stack of 3 Quadro RTX 8000 GPUs for the experiments in the paper. Training takes approximatively 20Gb of memory per GPU, when using a batch size of 8 and 2 gradient accumulation steps on each GPU.

**Comment by jd-yang-b** [comment]
Hi, thank you for the answer!
